plugins/lightspeed-backend/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@janus-idp/backstage-plugin-lightspeed-backend",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
once merged, you'll have to switch this to false in order to do releases to npmjs.com under the janus-idp namespace. See https://issues.redhat.com/browse/RHIDP-4029

Reply:
according to the discussion, see comments in https://issues.redhat.com/browse/RHIDP-4029
we decided to keep this plugin as private with `0.x` version, and will rely on manual releasing upon request.